pom.xml

```diff
@@ -307,16 +312,39 @@
                                     <include>**/*StructureTest.java</include>
                                 </includes>
                                 <runOrder>alphabetical</runOrder>
-                                <!-- <excludes>
-                                    <exclude>**/*</exclude>
-                                </excludes> -->
                                 <skipTests>${test.skip.tp}</skipTests>
                                 <systemPropertyVariables>
                                   <build.dir>${project.build.directory}</build.dir>
                                   <log4j.configuration>file:${project.build.directory}/test-classes/log4j.properties</log4j.configuration>
                                 </systemPropertyVariables>
                             </configuration>
-                       </execution>
+                        </execution>
+                        <execution>
+                            <id>codepipelines</id>
+                            <goals>
+                                <goal>test</goal>
+                            </goals>
+                            <phase>test</phase>
+                            <configuration combine.self="override">
+                                <reuseForks>false</reuseForks>
+                                <forkCount>1</forkCount>
+                                <parallel>none</parallel>
+                                <threadCount>1</threadCount>
+                                <perCoreThreadCount>false</perCoreThreadCount>
+                                <includes>
+                                    <include>${test.pipelines.includes}</include>
+                                </includes>
+                                <runOrder>alphabetical</runOrder>
+                                <excludes>
+                                    <exclude>${test.pipelines.excludes}</exclude>
+                                </excludes>
+                                <skipTests>${test.skip.pipelines}</skipTests>
+                                <systemPropertyVariables>
+                                    <build.dir>${project.build.directory}</build.dir>
+                                    <log4j.configuration>file:${project.build.directory}/test-classes/log4j.properties</log4j.configuration>
+                                </systemPropertyVariables>
+                            </configuration>
+                        </execution>
```

Comment:
Do the janusgraph-codepipeline-ci tests warrant this custom test configuration? Would default surefire configuration suffice?